src/main/java/org/rumbledb/runtime/flwor/FlworDataFrameUtils.java

```diff
@@ -409,7 +413,7 @@ public static Row reserializeRowWithNewData(
      * @param df - df to perform the operation on
      * @param offset - starting offset for the first index
      * @param indexName - name of the index column
-     * @return returns Dataset<Row> with the added 'indexName' column containing indices
```

Comment:
Javadocs were creating problems for mvn-javadoc-plugin during tests. Brackets are not allowed in javadocs with this format <>